### DISCUSSION TITLE
Should the default be to import the English local specifically?

### DISCUSSION THREAD

**Discussion by dijonkitchen** [faker-js/faker#887] [Q&A] [answered]
Based on https://fakerjs.dev/api/localization.html#individual-localization-packages, it seems like we're supposed to specify a locale for a smaller import. Since the default is English, should English users do the same? 

Something like: 

```js
import faker from '@faker-js/faker/locale/en';
```
instead of 
```js
import { faker } from '@faker-js/faker';
```

**Comment by Shinigami92** [comment, marked as answer]
If you know you don't need any other locales, yes, you can do that
But when using it just for tests, you wont recognize any impact, because you don't need to keep care of e.g. bundle size

**Reply by dijonkitchen** [reply]
Wouldn't you always want a smaller bundle size?

**Reply by Shinigami92** [reply]
I mean, it doesn't matter in a test environment when just executing unit or e2e tests, because you install the devDependencies anyway next to your tests :shrug:

**Reply by dijonkitchen** [reply]
Ah I see. That makes sense if it's a devDependency in npm. 

I'm thinking about using it as a normal dependency for a function as a service (AWS Lambda), so thinking it'll help with a smaller bundle.
